components/ide/code/leeway.Dockerfile

```diff
@@ -29,7 +29,7 @@ ENV PATH $NVM_DIR/versions/node/v$NODE_VERSION/bin:$PATH
 ENV PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD 1
 ENV ELECTRON_SKIP_BINARY_DOWNLOAD 1
 
-ENV GP_CODE_COMMIT eb4bf8291aa39a0d027a3de93fc63fe191db26e7
+ENV GP_CODE_COMMIT c95abe662e581ee6484c0b29d9a02b3b56a2f516
```

Comment:
FYI, this commit hasn't been merged to branch `gp-code` yet: https://github.com/gitpod-io/openvscode-server/pull/126 -- should we wait for a merge?

Reply:
yeah I have some thoughts on the changes in openvscode-server side that I want to double check with Anton first when he's back on Monday